packages/visx-tooltip/src/hooks/useTooltipInPortal.tsx

```diff
@@ -32,16 +34,22 @@ export type UseTooltipPortalOptions = Pick<PortalProps, 'zIndex'> & {
  */
 export default function useTooltipInPortal({
   detectBounds: detectBoundsOption = true,
+  portalContainer,
   zIndex: zIndexOption,
   ...useMeasureOptions
 }: UseTooltipPortalOptions | undefined = {}): UseTooltipInPortal {
   const [containerRef, containerBounds, forceRefreshBounds] = useMeasure(useMeasureOptions);
 
+  const portalContainerRect = useMemo(
```

Comment:
hmm, I don't think this will necessarily update the way we want, it's just computed once and if the container changes sizes or moves this will be stale/wrong.

this is the advantage of `useMeasure`, I'm a bit apprehensive of adding this if it's potentially going to result in issues of it being stale / incorrectly positioned.

Reply:
This was a really good point indeed! I was to simulate containers moving / changing sizes in the demo page and to confirm that my prior implementation didn't handle that gracefully.

Unfortunately `useMeasure` wouldn't work here because of the legacy type of ref it generates, which doesn't let us get the actual HTML element from the ref when we need it in `Portal`. That is a [known issue of this library](https://github.com/streamich/react-use/issues/1227) and it turns out that the library is not maintained anymore. In the issue, there are recommendations to use [@react-hookz/web](https://github.com/react-hookz/web) instead, but I couldn't find any way to target a specific hook in the library and importing the whole library just to use one hook didn't feel like a good idea.

Instead, I've opted for using [@react-hook/resize-observer](https://www.npmjs.com/package/@react-hook/resize-observer), and I was able to verify that my solution works as expected when containers move and change sizes.

This was before the fix, with randomly changing container positions and sizes:

https://user-images.githubusercontent.com/6173255/201192913-75db6a8a-8424-46f4-a5f4-247fa078dd09.mov

And this is after the fix:

https://user-images.githubusercontent.com/6173255/201193732-db609632-a3eb-4d4d-9054-0ea9ec91bbf1.mov